tools/openshift/ocp-ipi-aws/ocp-ipi-aws-prep/machine-set-resources.tf

```diff
@@ -52,6 +58,25 @@ data "template_file" "machine_set_template" {
   ]
 }
 
+# Render the template file.
+data "template_file" "machine_set_ha_template" {
+  template = file("${path.module}/templates/machine-set.yaml")
+
+  vars = {
+    az                   = data.aws_subnet.target_public_subnet_ha.availability_zone
```

Comment:
Hmm, but we need to use different AZs for each, right?

Reply:
With this we are taking AZ from a different public subnet, as the public subnet is per availability zone, they will use different zones.

Reply:
Oh,  I didn't see the input anywhere, is that provided by OCP or AWS?

Reply:
We get it from the terraform state , so should be provided by AWS.